src/pudl/analysis/state_demand.py

```diff
@@ -267,8 +267,8 @@ def load_ventyx_hourly_state_demand(path: str) -> pd.DataFrame:
 @multi_asset(
     compute_kind="Python",
     outs={
-        "raw_hourly_demand_matrix_ferc714": AssetOut(),
-        "utc_offset_ferc714": AssetOut(),
+        "_out_ferc714__hourly_pivoted_demand_matrix": AssetOut(),
```

Comment:
I didn't want to use `raw` in the name because it's kind of a keyword in the new naming convention. Seems like the primary transformation here was a pivot.

Reply:
Seems fine!